pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/jstl/DisabledInvocationBeanResolver.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.pulsar.functions.transforms.jstl;
+
+import jakarta.el.BeanELResolver;
+import jakarta.el.ELContext;
+import jakarta.el.MethodNotFoundException;
+
+public class DisabledInvocationBeanResolver extends BeanELResolver {
+  @Override
+  public Object invoke(
+      ELContext context, Object base, Object method, Class<?>[] paramTypes, Object[] params) {
+    throw new MethodNotFoundException("Method invocations are disabled: " + method);
```

Comment:
great idea !


Maybe it would be useful to add a comment to explain why we are doing so